scripts/process-model-data.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+"""Process CSV model data, e.g. for aggregation or merging
+
+Usage:
+    scripts/process-model-data.py [-h] <query-name> <csv-dir>
+
+Options:
+    -h --help       Show this screen.
+"""
+import sys
+sys.path.insert(0, '.')
+
+from docopt import docopt
+import pandas as pd
+import os.path
+
+from dmscripts.models import queries
+
+
+CONFIG = {
```

Comment:
I don't think this really achieves the idea of separating configuration from the code. Modifying this structure doesn't allow us to:
* add/remove output fields
* add new output files that are joining different CSVs
* change the count logic for one-to-many joins

So for example if we want to add a file that shows how many briefs each supplier has applied for we'd have to write a new separate function, even though all the logic behind it is more or less the same.